pkgs/tools/misc/flowgger/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, rustPlatform
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, openssl
+, capnproto
+, CoreServices
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "flowgger";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
```

Comment:
How about using `fetchCrate` here?
I don't think we need to maintain Cargo.lock ourselves with using it.

Reply:
It is a good idea. I changed source file